src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -692,6 +695,10 @@ protected DynamoModel(IStartConfiguration config)
             OnRequestUpdateLoadBarStatus(new SplashScreenLoadEventArgs(Resources.SplashScreenInitPreferencesSettings, 30));
 
             PreferenceSettings = (PreferenceSettings)CreateOrLoadPreferences(config.Preferences);
+
+            //Set the DynamoModel.PreferenceSetting as a static reference to the PreferenceSetting class for use in PreferenceSettings.Instance
+            PreferenceSettings.dynamoModelRuntimePreferenceSettings = PreferenceSettings;
```

Comment:
This is for covering the case that we are creating a default preferences which was not saved yet, right?